README.md

```diff
@@ -108,18 +108,16 @@ in security breach "dangerous") side-effects.
 
 ### ``failure``
 
-Whether this check decided that the job matrix failed.
-
+This check determines whether the job matrix has failed and returns the status as `success`.
 
 ### ``result``
 
-Failure or success result of the job matrix.
+returns `failure` or `success` as result of the job matrix.
```

Comment:
It's probably worth noting that these are strings and make the sentence start with a capital letter.
```suggestion
Returns `'failure'` or `'success'` as result of the job matrix.
```